lib/index.js

```diff
@@ -27,7 +27,7 @@ module.exports.pitch = function pitch(remainingRequest) {
   // builds in different directories.
   const newRequestPath = remainingRequest.replace(
     this.resourcePath,
-    `.${path.sep}${path.relative(this.context, this.resourcePath)}`
+    `.${path.sep}${path.relative(this.rootContext, this.resourcePath)}`
```

Comment:
Can we use `this.rootContext || this.context` to avoid breaking change?

Reply:
I think we can )

Reply:
Can you view also https://github.com/webpack-contrib/val-loader/pull/37 ?

Reply:
@Delagen in my todo, release will be in this week

Reply:
Moved to https://github.com/webpack-contrib/expose-loader/pull/92